pydarn/plotting/grid.py

```diff
@@ -178,11 +160,9 @@ def plot_grid(cls, dmap_data: List[dict], record: int = 0,
             warnings.simplefilter("ignore")
             for stid in dmap_data[record]['stid']:
                 _, aacgm_lons, _, _, ax =\
-                        Fan.plot_fov(stid, dtime,
-                                     boundary=fov,
+                        Fan.plot_fov(stid, date,
                                      lowlat=lowlat,
                                      line_color=line_color,
-                                     fov_color=fov_color,
                                      radar_location=radar_location,
                                      radar_label=radar_label,
                                      ax=ax)
```

Comment:
```suggestion
                                     ax=ax, **kwargs)
```
Sorry it made me split it up around the deleted line to be able to do it as a suggestion.
ax=ax still needs to be here as if it's a combined grid file it will make a new ax for each radar instead of putting them all together.